cypress/fixtures/inboxNotif-users.fixture.json

```diff
@@ -0,0 +1,19 @@
+{
+  "cyUserMemmerAPI": {
+    "clientID": "APP-6RTM54FDADENEKUK",
+    "clientSecret": "95533ab6-3594-40a7-98b8-c8db637f2514",
+    "orcidMember": "0000-0003-0667-1019",
+    "bearer": "2900e981-b9d3-488b-9106-ed70a241d54b"
+  },
+  "cyNotifPerm": {
+    "oid": "0000-0002-5078-7150",
+    "password": "test1234",
+    "clientBearer": "a4320bf6-868b-4007-a2fa-aa534b6083f1",
+    "curlPermissionNotifPath": "@/Users/karenmadrigal/Documents/projects/registry/cypress/fixtures/permission_notification.fixture.json",
```

Comment:
we can use a 'relative path' so is easier to run later on another computer

Reply:
Fixed, thank you!